### PR TITLE
solved challenge

### PR DESCRIPTION
**Pull request by micahThor**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution

### CODE REVIEW

File: tree/src/main/java/tree/Tree.java [hunk 1]
```diff
@@ -1,4 +1,59 @@
...
+    public ArrayList<Integer> preOrder() {
+        ArrayList<Integer> intList = new ArrayList<>();
+        return preOrder(root, intList);
+    }
+
+    private ArrayList<Integer> preOrder(Node node, ArrayList<Integer> list) {
+
+        if (node != null) {
+            list.add(node.value);
+            inOrder(node.left, list);
```
Comment (JamesDansie) [inline tree/src/main/java/tree/Tree.java:23]: this should call preOrder, not inOrder

File: tree/src/main/java/tree/Tree.java [hunk 2]
```diff
@@ -1,4 +1,59 @@
...
+    public ArrayList<Integer> postOrder() {
+        ArrayList<Integer> intList = new ArrayList<>();
+        return postOrder(root, intList);
+    }
+
+    private ArrayList<Integer> postOrder(Node node, ArrayList<Integer> list) {
+
+        if (node != null) {
+            inOrder(node.left, list);
+            inOrder(node.right, list);
```
Comment (JamesDansie) [inline tree/src/main/java/tree/Tree.java:54]: This should call postOrder, not inOrder
